addons/l10n_fr_account/models/template_fr.py

```diff
@@ -9,6 +9,7 @@ class AccountChartTemplate(models.AbstractModel):
     @template('fr')
     def _get_fr_template_data(self):
         return {
+            'name': _('Plan comptable général'),
```

Comment:
The text to translate should be in English (see the related comment in the .pot file).